gapis/service/path/path.proto

```diff
@@ -428,4 +428,6 @@ message Thumbnail {
         Command command = 5;
         CommandTreeNode command_tree_node = 6;
     }
+
+    bool disableOptimization = 7;
```

Comment:
Maybe use ReplaySettings here too? Might require moving it to this package though.